README.md

```diff
@@ -11,7 +11,7 @@ workflow "Example Workflow" {
 }
 
 action "Lint" {
-  uses = "lgeiger/black-action@master"
+  uses = "Dquentin/black-action@master"
```

Comment:
Shouldn't this be reverted to `"lgeiger/black-action@master"`, since this is the "original" repository that will be used as action.